src/data/ores/bronzeOre.js

```diff
@@ -0,0 +1,16 @@
+import { itemType } from '../../enums'
+
+const { freeze } = Object
+
+/**
+ * @property farmhand.module:items.bronzeOre
+ * @type {farmhand.item}
+ */
+export const bronzeOre = freeze({
+  description: 'A piece of bronze ore you dug up in the field.',
+  doesPriceFluctuate: true,
```

Comment:
I hadn't really thought about the ores having fluctuating price points, but this is a really interesting idea! We may need to revisit some semantics within the game's text content, as it's implied that only crops have fluctuating prices.

Reply:
actually, you did spec this :) 👉  https://github.com/jeremyckahn/farmhand/issues/135

but i'm cool with whatever! i thought it was a great idea, but did not consider copy

Reply:
D'oh! 🤦‍♂️

Silly me, I can't remember my own ideas. Good thing we have things captured in tickets for reference! 😅

Obviously this is fine to leave in. Sorry for the back and forth on that!